rocAL/rocAL_pybind/README.md

```diff
@@ -20,7 +20,7 @@ rocAL_pybind installs during [MIVisionX build](https://github.com/GPUOpen-Profes
 
 * Install pip packages
 ````
-pip3 install numpy opencv-python torch
+pip3 install numpy opencv-python torch pillow
```

Comment:
@fiona-gladwin Seems like capital letter 'P'. Did you try this?
https://pypi.org/project/Pillow/

Reply:
Both pillow and Pillow installs the same pip package